python/llm/dev/benchmark/all-in-one/save_openvino.py

```diff
@@ -0,0 +1,88 @@
+from transformers import AutoTokenizer
+from optimum.intel import OVWeightQuantizationConfig
+from optimum.intel.openvino import OVModelForCausalLM
+
+import os
+from pathlib import Path
+import argparse
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument('-h',
+                        '--help',
+                        action='help',
+                        help='Show this help message and exit.')
+    parser.add_argument('-m',
+                        '--model_id',
+                        default='Qwen/Qwen1.5-0.5B-Chat',
+                        required=False,
+                        type=str,
+                        help='orignal model path')
+    parser.add_argument('-p',
+                        '--precision',
+                        required=False,
+                        default="int4",
+                        type=str,
+                        choices=["fp16", "int8", "int4"],
+                        help='fp16, int8 or int4')
+    parser.add_argument('-o',
+                        '--output',
+                        required=False,
+                        type=str,
+                        help='path to save the ir model')
+    parser.add_argument('-ms',
+                        '--modelscope',
+                        action='store_true',
+                        help='download model from Model Scope')
```

Comment:
Please deal with the model-id as we have done in `save.py`
And add parameter for grou_size (e.g. 64, 128, etc.)